lib/travis/build/script/julia.rb

```diff
@@ -58,31 +58,38 @@ def setup
         def announce
           super
 
-          sh.cmd "julia -e 'versioninfo()'"
+          sh.cmd 'julia --color=yes -e "VERSION >= v\"0.7.0-DEV.3630\" && using InteractiveUtils; versioninfo()"'
           sh.echo ''
         end
 
         def script
           sh.echo 'Executing the default test script', ansi: :green
-          set_jl_pkg
+
+          # Extract the package name from the repository slug (org/pkgname.jl)
+          m = /(\w+)\/(\w+)(?:\.jl)?(?:\.git)?$/.match('${TRAVIS_REPO_SLUG}')
+          if m != nil
+            sh.export 'JL_PKG', m[1]
+          end
```

Comment:
Still no good. It looks like this match is failing.
Any idea why this isn't matching?